Controllers/OperationController.cs

```diff
@@ -16,18 +17,45 @@ namespace ARMExplorer.Controllers
     public class OperationController : ApiController
     {
         private readonly IArmRepository _armRepository;
+        private static readonly MemoryCache SwaggerCache = new MemoryCache("SwaggerDefinitionCache");
```

Comment:
What do we gains from using MemoryCache over just a plain Dictionary? Is it mostly about the ability to kick things out automatically if there is memory pressure? Given that the max number of specs is static and not extremely large, I'm not convinced it's worth the extra complexity. I find the whole `AddOrGetExisting` / Lazy pattern below a little non-obvious.